import catchError from "../utils/catchError";


export const registerHandler=catchError(
    async (req,res)=>{
        
    }
)
